translations/ENCv311_to_OGR.js

```diff
@@ -565,11 +565,12 @@ enc311 = {
     if (enc311.configOut == undefined)
     {
       enc311.configOut = {};
-      enc311.configOut.OgrDebugDumptags = hoot.Settings.get('ogr.debug.dumptags');
-      enc311.configOut.OgrDebugDumpvalidate = hoot.Settings.get('ogr.debug.dumpvalidate');
+      enc311.configOut.OgrDebugDumptags = (hoot.Settings.get('ogr.debug.dumptags') === 'true');
+      enc311.configOut.OgrDebugDumpvalidate = (hoot.Settings.get('ogr.debug.dumpvalidate') === 'true');
+      enc311.configOut.OgrThrowError = (hoot.Settings.get('ogr.throw.error') === 'true');
```

Comment:
I'm not sure this has the desired effect.  `true === 'true'` is `false`.
If this PR is converting appropriate settings to boolean everywhere I don't think any conditional is needed.

Reply:
From what I can find, the core is sending the boolean values as strings.
E.g. Running this in a Node script:
```
console.log('');
console.log('Raw debug value: ' + hoot.Settings.get("ogr.debug.dumptags"));
console.log('Raw debug type: ' + typeof hoot.Settings.get("ogr.debug.dumptags"));

if (hoot.Settings.get("ogr.debug.dumptags"))
{
  console.log(' debug is True');
}
else
{
  console.log(' debug is False');
}


// LOTS of debug output
hoot.Settings.set({"ogr.debug.dumptags":true});

console.log('');
console.log('After Change value: ' + hoot.Settings.get("ogr.debug.dumptags"));
console.log('After change type: ' + typeof hoot.Settings.get("ogr.debug.dumptags"));

if (hoot.Settings.get("ogr.debug.dumptags"))
{
  console.log(' debug is True');
}
else
{
  console.log(' debug is False');
}
```

Gives this output:
```
Raw debug value: false
Raw debug type: string
 debug is True

After Change value: true
After change type: string
 debug is True
```

The code I put in the variable assignment tries to convert the string to a boolean so that the `if ( Setting )...` checks would work.

Reply:
So `hoot.Settings.set({"writer.thematic.structure":false});`
leads to `hoot.Settings.get("writer.thematic.structure");` returning `'false'`?

If that's the case the setter should just use a string everywhere.